### ISSUE TITLE
Accessibility mode should work on iOS

### ISSUE THREAD

**Issue by devinprater** [libretro/RetroArch#16532]
# First and foremost consider this:
- Only RetroArch bugs should be filed here. Not core bugs or game bugs
- This is not a forum or a help section, this is strictly developer oriented

## Description

When using Retroarch for iOS (released on the App Store on May 15, 2024), accessibility mode, where the menu items are read aloud, does not work.

### Expected behavior

When setting accessibility-mode to "true" in the retroarch.cfg, or when launching Retroarch with VoiceOver active, Retroarch should say "Retroarch accessibility on", and then speak what the person moves to using a gamepad, or with their finger (direct touch).

### Actual behavior

Retroarch says nothing, and VoiceOver detects nothing on the screen.

### Steps to reproduce the bug

1. [Turn on VoiceOver](https://developer.apple.com/documentation/uikit/accessibility_for_uikit/supporting_voiceover_in_your_app).
2. Open Retroarch.
3. Move your finger around the screen, or press the D-Pad buttons on a gamepad.

### Bisect Results

This always happened on Retroarch for iOS.

### Version/Commit
You can find this information under Information/System Information

I cannot get this info since I cannot use the app.

### Environment information

- OS: iOS 17.5

**Comment by devinprater** [comment]
VoiceOver [can be checked to see if it's on](https://developer.apple.com/documentation/swiftui/accessibilitytechnologies/voiceover). This can be used to automatically turn on accessibility mode. Accessibility mode speech output can then be [posted as an accessibility notification](https://developer.apple.com/documentation/uikit/uiaccessibility/notification). Alternatively, the [AV Speech synthesizer](https://developer.apple.com/documentation/avfoundation/speech_synthesis/) can be used to read output, a bit like on MacOS.

**Comment by warmenhoven** [comment]
Yeah on macOS it invokes `say` directly rather than call the APIs. It's on my list to fix.
